tests/resolvers/Mutation/createEvent.spec.ts

```diff
@@ -159,76 +168,187 @@ describe("resolvers -> Mutation -> createEvent", () => {
+  //     })
+  //   );
+
+  //   const recurringEvents = await Event.find({
+  //     recurrance: "ONCE",
+  //   }).lean();
+
+  //   expect(recurringEvents).toBeDefined();
+  //   expect(recurringEvents).toHaveLength(1);
+
+  //   const attendeeExists = await EventAttendee.exists({
+  //     userId: testUser!._id,
+  //     eventId: createEventPayload!._id,
+  //   });
+
+  //   expect(attendeeExists).toBeTruthy();
+
+  //   const updatedTestUser = await User.findOne({
+  //     _id: testUser?._id,
+  //   })
+  //     .select(["eventAdmin", "createdEvents", "registeredEvents"])
+  //     .lean();
+
+  //   expect(updatedTestUser).toEqual(
+  //     expect.objectContaining({
+  //       eventAdmin: expect.arrayContaining([createEventPayload?._id]),
+  //       createdEvents: expect.arrayContaining([createEventPayload?._id]),
+  //       registeredEvents: expect.arrayContaining([createEventPayload?._id]),
+  //     })
+  //   );
+  // });
+
+  // it(`creates the Weekly recurring event and returns it`, async () => {
+  //   await User.updateOne(
+  //     {
+  //       _id: testUser?._id,
+  //     },
+  //     {
+  //       $push: {
+  //         createdOrganizations: testOrganization?._id,
+  //         joinedOrganizations: testOrganization?._id,
+  //       },
+  //     }
+  //   );
+
+  //   const args: MutationCreateEventArgs = {
+  //     data: {
+  //       organizationId: testOrganization?.id,
+  //       allDay: false,
+  //       description: "newDescription",
+  //       endDate: new Date("2023-01-29T00:00:00Z"),
+  //       endTime: new Date().toUTCString(),
+  //       isPublic: false,
+  //       isRegisterable: false,
+  //       latitude: 1,
+  //       longitude: 1,
+  //       location: "newLocation",
+  //       recurring: true,
+  //       startDate: new Date("2023-01-01T00:00:00Z"),
+  //       startTime: new Date().toUTCString(),
+  //       title: "newTitle",
+  //       recurrance: "WEEKLY",
+  //     },
+  //   };
+
+  //   const context = {
+  //     userId: testUser?.id,
+  //   };
+  //   const { createEvent: createEventResolver } = await import(
+  //     "../../../src/resolvers/Mutation/createEvent"
+  //   );
+
+  //   const createEventPayload = await createEventResolver?.({}, args, context);
+
+  //   expect(createEventPayload).toEqual(
+  //     expect.objectContaining({
+  //       allDay: false,
+  //       description: "newDescription",
+  //       isPublic: false,
+  //       isRegisterable: false,
+  //       latitude: 1,
+  //       longitude: 1,
+  //       location: "newLocation",
+  //       recurring: true,
+  //       title: "newTitle",
+  //       creator: testUser?._id,
+  //       admins: expect.arrayContaining([testUser?._id]),
+  //       organization: testOrganization?._id,
+  //     })
+  //   );
+
+  //   const recurringEvents = await Event.find({
+  //     recurring: true,
+  //     recurrance: "WEEKLY",
+  //   }).lean();
+
+  //   expect(recurringEvents).toBeDefined();
```

Comment:
Why hasn't this code been restored given the past comments by me and @sumitra19jha ?

Reply:
sorry , I removed the commit due to some format issues.